OasisDEX/oasis-borrow
# Ajna base support

# [Ajna base support](https://app.shortcut.com/oazo-apps/story/11979/ajna-support-for-base)

<please insert a shortcut link above>
  
## Changes 👷‍♀️
  <Please add short list of changes>

- removed dependency from context in places where we can use network from URL
  
## How to test 🧪
  <Please explain how to test your changes>

- everything should work as usual including aave, ajna, spark, maker, dsr etc.
- it should be possible to go to ajna open position (borrow, earn, multiply) on base
